pywwt/nbextension/static/wwt_json_api.js

```diff
@@ -182,10 +182,36 @@ function wwt_apply_json_message(wwt, msg) {
       wwt.setBackgroundImageByName(msg['mode']);
       wwt.setForegroundImageByName(msg['mode']);
       break;
-
+/*
     case 'track_object':
       wwtlib.WWTControl.singleton.renderContext.set_solarSystemTrack(msg['code']);
       break;
+*/
+    case 'track_and_zoom':
+      var object = msg['obj'];
+      var ra = wwt.getRA();
+      var dec = wwt.getDec();
```

Comment:
I don't understand why the current RA/Dec are relevant. What do those mean in the context of the planet/body views? Is it related to the direction from which one is viewing the solar system? It would be good to add some comments here to explain it.

Reply:
I get them a) because RA and dec are arguments of `Place.create()`, and b) so RA/dec remain the same if you're tracking the Sun. (If you're not tracking the Sun, then the current RA/dec do not actually influence the final orientation of the object due to #177 -- as far as I can tell.)

Given that the current RA/Dec doesn't seem to matter when creating a `Place`, would you rather assign a fixed RA/dec for the Sun and just pass fixed, pre-selected RA/dec values to `Place.create()`?

Reply:
I don't really have a strong preference, but I think that whatever you do you should add a comment to explain why you are doing it